src/algorand_transactions/transaction_json.rs

```diff
@@ -77,6 +77,9 @@ pub struct AlgorandTransactionJson {
     pub key_reg_transaction: Option<KeyRegTransactionJson>,
 
     pub id: Option<String>,
+
+    #[serde(rename = "inner-txns")]
+    pub inner_txs: Option<Vec<AlgorandTransactionJson>>,
```

Comment:
Is there a reason you dropped the `n`?

Reply:
Because in other places we refereed to transactions as `txs`. It's a thing I considered

Reply:
Discussed privately & resolved.